Core/clim-basic/regions.lisp

```diff
@@ -90,8 +90,12 @@
 
 ;;; coordinate is defined in coordinates.lisp
 
-(defvar +everywhere+ (make-instance 'everywhere-region))
-(defvar +nowhere+ (make-instance 'nowhere-region))
+(defconstant +everywhere+ (make-instance 'everywhere-region))
+(defconstant +nowhere+ (make-instance 'nowhere-region))
+
+(setf everywhere* +everywhere+)
+(setf nowhere* +nowhere+)
+
 
```

Comment:
I don't get this change. Why can't we use `+everywhere+` and `+nowhere+` from the very beginning?

NB `defconstant` in sbcl is patched anyway to `defvar` for sake of getting rid of problems when we recompile things.